### ISSUE TITLE
Downlod doubt

### ISSUE THREAD

**Issue by balajik2207** [ultralytics/hub#589]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

If I train the model can I dowload and use it.

**Comment by kalenmike** [comment]
@balajik2207 Yes, when your model is trained you can download it in PyTorch format or export it to any of our supported formats and download it.

You can also download your model to your phone using our App and preview in real-time.
